aspnetcore/fundamentals/websockets.md

```diff
@@ -148,13 +139,42 @@ When accepting the WebSocket connection before beginning the loop, the middlewar
 
 ::: moniker range=">= aspnetcore-2.2"
 
-### Handle client disconnects
+## WebSocket lifetime
+
+The WebSocket that is received from `AcceptWebSocketAsync` lives only as long as the middleware pipeline for the request is still running. When the middleware pipeline ends, the socket is closed. If your code continues to try to send and receive messages, a `WebSocketException` exception is raised with the following message:
+
+> The remote party closed the WebSocket connection without completing the close handshake.
+
+The exception has an `ObjectDisposedException` inner exception with the following message:
+
+> Cannot write to the response body, the response has completed. Object name: 'HttpResponseStream'.
+
+To avoid this error, don't let the request pipeline end before you're finished using the WebSocket. If you use a WebSocket from outside the request context (in a background thread or from another request) use code like the following example to make sure the request pipeline doesn't end prematurely:
```

Comment:
> If you use a WebSocket from outside the request context (in a background thread or from another request) use code like the following example to make sure the request pipeline doesn't end prematurely:

That's too long. Shorten if possible. Maybe

When using a WebSocket from outside the request context:

* For example, in a background thread or from another request.
* Use code like the following to make sure the request pipeline doesn't end prematurely: